### PR TITLE
update deployment specialized knowledge text

### PR DESCRIPTION
**Pull request by sourabh3b**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by sourabh3b** [comment]
@gargana Please take a look at this PR as per Sheryl's request. thanks

**Comment by gargana** [comment]
I will be reviewing with the tech writer team tomorrow AM.

File: docs/partner_editable/specialized_knowledge.adoc [hunk 1]
```diff
@@ -2,4 +2,4 @@
 // For example: “familiarity with basic concepts in networking, database operations, and data encryption” or “familiarity with <software>.”
 // Include links if helpful. 
 // You don't need to list AWS services or point to general info about AWS; the boilerplate already covers this.
-This deployment guide was designed for general purpose use and requires a moderate level of familiarity with the Aerospike database and AWS services.  To learn more about high performance and availability configurations please refer to Aerospike recommendations for https://docs.aerospike.com/docs/deploy_guides/aws/recommendations/index.html[AWS instance types with locally attached SSDs^].
\ No newline at end of file
+There are many considerations for a production-grade Aeropsike deployment.  This Quick Start was designed for general purpose use and is not intended as a high performance configuration.  To learn more about high performance and fully resilient configurations, please refer to Aerospike best practices for https://docs.aerospike.com/docs/deploy_guides/aws/recommendations/index.html[AWS instance types with locally attached SSDs^].
```
Comment (MarciaRieferJohnston) [inline docs/partner_editable/specialized_knowledge.adoc:5]:
```suggestion
There are many considerations for a production-grade Aerospike deployment. This Quick Start was designed for general-purpose use and is not intended as a high-performance configuration. To learn more about high-performance and fully resilient configurations, see Aerospike best practices for AWS instance types with locally attached SSDs as documented here: https://docs.aerospike.com/docs/deploy_guides/aws/recommendations/index.html[Aerospike on Amazon EC2 - Recommendations^].
```

**Review by gargana — APPROVED** [review]
Confirmed merge conflict changes. Shipping

**Pull request merged**